zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java

```diff
@@ -83,7 +84,13 @@ public static TensorFlowModel create(final URI modelResource,
   public static TensorFlowModel create(final Model.Id id,
                                        final URI modelResource,
                                        final Options options) throws IOException {
-    final URI localDir = FileSystemExtras.downloadIfNonLocal(modelResource);
+    // GCS requires that directory URIs have a trailing slash, so add the slash if it's missing
+    // and the URI starts with 'gs'.
+    final URI normalizedUri =
+        !CloudStorageFileSystem.URI_SCHEME.equalsIgnoreCase(modelResource.getScheme())
```

Comment:
This is using a transitive dependency this should be abstracted in `zoltar-core`